.circleci/config.yml

```diff
@@ -13,14 +13,16 @@ jobs:
           command: "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/latest/download/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
       - run:
           name: Build latest earthly
-          command: earthly --use-inline-cache +for-linux
+          # circle CI is being run as user "circleci" with SUDO_USER="root"; however the circleci user does not have access to /root/.earthly/config.yml
+          # therefore, we must unset the SUDO_USER; otherwise we will get this error: "open /root/.earthly/config.yml: permission denied"
+          command: SUDO_USER="" earthly --use-inline-cache +for-linux
       - run:
           name: Bootstrap
           command: sudo ./build/linux/amd64/earthly bootstrap
       - run:
           name: Run tests
           command: |-
-              ./build/linux/amd64/earthly --ci -P \
+              SUDO_USER="" ./build/linux/amd64/earthly --ci -P \
```

Comment:
Do we need to update our docs to encourage our users to do this too?

Reply:
If that's the case, then maybe we should just remove the logic that makes use of `SUDO_USER`, or we can fine-tune it to only work when the current user is root and the `SUDO_USER` is not root, then the need for circle-ci-specific entry is removed.